packages/web-components/fast-components/src/accordion-item/index.ts

```diff
@@ -14,12 +13,11 @@ import { AccordionItemStyles as styles } from "./accordion-item.styles";
  * @remarks
  * HTML Element: \<fast-accordion-item\>
```

Comment:
@eisenbergeffect @nicholasrice thoughts on the comment syntax and what we should include for these?

Reply:
@nicholasrice How should we handle the comments in this case? We probably needs a new comment for the function that says something like "Creates a component registration for `fast-accordion-item` usable with the `DesignSystem`." Thoughts?

Reply:
I think that works, but I'd make sure that the `DesignSystem` reference is an `{@link @micorosoft/fast-foundation:DesignSystem}` so that our docs are referential. Or perhaps we have a link to a new tutorial-like section on how to use these definitions. Maybe both?

Reply:
Leaving these for a follow-up PR - but happy to make the updates